packages/@rnw-scripts/doxysaurus/src/logger.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+//
+// The logger used by the Doxysaurus project.
+//
+// The goal is make the logging looking simpler in the code.
+// - 'Warning:' prefixes are rendered in yellow color.
+// - 'Error:' prefixes are rendered in red color.
+// - Text in square brackets [] is rendered in green.
+// - Text in curly brackets {} is rendered in bright cyan.
```

Comment:
Consider having prefixes as an official part of the API. That lets view representation be decoupled from logging caller.

Reply:
I have added the `log.warning` and 'log.error'. As for the [] and {} colors I would like to keep it to keep the logger calls simple.